content/intro-to-storybook/vue/pt/conclusion.md

```diff
@@ -3,31 +3,32 @@ title: 'Conclusão'
 description: 'Fusão de todo o conhecimento adquirido e aprendizagem de técnicas extra de Storybook'
 ---
 
-Parabéns! Acabaste de criar o teu primeiro interface de utilizador com Storybook. Durante esta jornada aprendeste como construir, compor, testar e implementar componentes de interface de utilizador.
-Se estiveste a seguir o tutorial, o repositório e o Storybook implementado irá ser idêntico a este:
+Parabéns! Você criou sua primeira interface do usuário no Storybook. Ao longo do caminho, você aprendeu a criar, compor, testar e implantar componentes da interface do usuário. Se você seguiu o tutorial, seu repositório e o Storybook implantado devem ficar assim:
 
 [📕 **GitHub repo: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
 [🌎 **Storybook implementado**](https://clever-banach-415c03.netlify.com/)
 
-O Storybook é uma ferramenta bastante poderosa para React, Vue, Angular.
-É composta por uma comunidade de desenvolvimento próspera e um número grande de extras. Esta introdução somente arranha a superfície das potencialidades existentes. Estamos confiantes que depois da adoção do Storybook, irá ficar impressionado com o quanto é produtivo a construção de interfaces de utilizador duradouros.
+O Storybook é uma ferramenta poderosa para React, Vue e Angular.
+Possui uma comunidade de desenvolvedores próspera e uma grande variedade de complementos. Esta introdução arranha a superfície do que é possível. Estamos confiantes de que, ao adotar o Storybook, você ficará impressionado com a produtividade da criação de UIs duráveis.
```

Comment:
A better wording for this could be something like:
"O Storybook é uma ferramenta bastante poderosa pra React, Vue e Angular.
Possui uma comunidade de programadores próspera e uma grande variedade de extras. Esta introdução arranha a superfície do que é possível fazer. Estamos confiantes que ao adotar o Storybook, ficará impressionado com o quão produtivo é construir IUs  duradouros". 

Reasoning behind this:
- Left in "bastante" as a means to enphasize the power of the tooling. Leaving it in doesn't break the contents or grammar used. More of a embellishment to be perfectly honest.
- Avoid usage of "desenvolvedores" as it's not accurate for european portuguese, instead the usage of programadores is a common ground that both readers will understand.
- Avoid usage of "complementos" as it's connotation is a bit off in europe and might mislead the reader, Using "extras" reaches a middle ground and both readers understand it.
- "The usage of "duradouros" and "duráveis"  is accepted by both variations of the language, leaving in the original will not break the intended content.
- Using "IU" as a means to refer to "user interfaces" is probably a good choice, because at this stage of the tutorial the reader despite being new or a veteran as seen the words so many times and will know what it means.